examples/02_meta-analyses/13_plot_sdm.py

```diff
@@ -0,0 +1,207 @@
+"""
+
+.. _metas_sdm:
+
+==========================================
+Seed-based d Mapping (SDM) Meta-Analysis
+==========================================
+
+A tutorial on using SDM and SDM-PSI algorithms in NiMARE.
+
+This tutorial demonstrates how to perform meta-analysis using Seed-based d Mapping (SDM),
+including both the basic SDM estimator and the advanced SDM-PSI (Permutation of Subject Images)
+implementation with multiple imputation and Rubin's rules.
+
+SDM is a hybrid meta-analysis method that can accept both peak coordinates and
+whole-brain statistical images, preferentially using images when available.
+"""
+
+###############################################################################
+# Load Dataset
+# -----------------------------------------------------------------------------
+# .. note::
+#   The data used in this example come from a collection of NIDM-Results packs
+#   downloaded from Neurovault collection 1425, uploaded by Dr. Camille Maumet.
+import os
+from pprint import pprint
+
+from nilearn.plotting import plot_stat_map
+
+from nimare.correct import FWECorrector
+from nimare.dataset import Dataset
+from nimare.meta import SDM, SDMPSI, SDMKernel
+from nimare.utils import get_resource_path
+
+dset_file = os.path.join(get_resource_path(), "nidm_pain_dset.json")
+dset = Dataset(dset_file)
+
+# Split dataset for demonstration
+dset1 = dset.slice(dset.ids[:10])
+
+###############################################################################
+# Basic SDM Meta-Analysis
+# -----------------------------------------------------------------------------
+# The basic SDM estimator performs coordinate-based meta-analysis using an
+# anisotropic Gaussian kernel. When images are available, it uses them directly.
+#
+# **Key Features:**
+#
+# - Hybrid input support (coordinates + images)
+# - Anisotropic Gaussian kernel (FWHM=20mm default)
+# - Simple mean-based meta-analysis
+# - Returns stat, z, p, and dof maps
+
+meta = SDM()
+results = meta.fit(dset1)
+
+# Plot the results
+plot_stat_map(
+    results.get_map("z"),
+    cut_coords=[0, 0, -8],
+    draw_cross=False,
+    cmap="RdBu_r",
+    symmetric_cbar=True,
+    vmax=3,
+    title="SDM: Z-statistic Map",
+)
+
+print("\\nSDM Description:")
+pprint(results.description_)
+print("\\nSDM References:")
+pprint(results.bibtex_)
+
+###############################################################################
+# Custom Kernel Configuration
+# -----------------------------------------------------------------------------
+# You can customize the SDM kernel's full-width at half-maximum (FWHM)
+# parameter to control the smoothness of the reconstructed maps from coordinates.
+
+meta_custom = SDM(SDMKernel, kernel__fwhm=15)
```

Comment:
You do not need to include the SDMKernel object, you can just use `kernel__fwhm=15`